hackmcgill/dashboard
# Feat/888 invite users

### Tickets:

- #888 

### List of changes:

- feat: create a barebones invite page
- feat: add 'invite' item in navbar, and clean up navbar code
- feat: Add Toast HOC to Invite page
- feat: Add 'getInvites' endpoint to API library
- feat: Add 'ExistingInvites' table which shows all of the invites that were sent out
- feat: Wire in the existing invites table into the InvitePage

### Type of change:

Please delete options that aren't relevant.


### How did you do this?

### How to test:

- Log into an admin account.
- Invite another user to create an account (please try with all relevant user types. I tested only with admin).
- Go through user creation flow. Make sure that the user has the right permissions. They should not be redirected to the confirm account page because they were invited.

### Questions:

### PR Checklist:


### Screenshots:

![image](https://user-images.githubusercontent.com/17463268/97812060-be41af00-1c4c-11eb-8147-64136e5375ae.png)
